### ISSUE TITLE
onChange, react native, assign

### ISSUE THREAD

**Issue by poyannabati** [erikras/redux-form#3238]
### Are you submitting a **bug report** or a **feature request**?

Bug

### What is the current behavior?

Trying to set onChange prop on <Field /> on react native renders a RSOD as such; 

![image](https://user-images.githubusercontent.com/1499050/28518648-e48e6f56-7068-11e7-96f4-f59f46c12400.png)

```
            <Field
              name="model"
              component={TextField}
              onChange={(event, newValue, oldValue) => {
                console.log(event);
              }}
            />
```

### What's your environment?

6.80, RN 0.45.1

**Comment by beornborn** [comment]
@erikras could you take a look? it's critical

**Comment by beornborn** [comment]
https://github.com/erikras/redux-form/issues/3153

**Comment by beornborn** [comment]
related https://github.com/erikras/redux-form/issues/3153

**Comment by jkomyno** [comment]
Same for me

**Comment by AlimovSV** [comment]
@jkomyno PR?

**Comment by jkomyno** [comment]
@AlimovSV sure, but luckily I've discovered a cleaner way to handle this, so I'm going to drop that commit and change it a tiny bit. I'll reference this issue both in the commit log and in the PR message so you'll get notified by it.

**Comment by AlimovSV** [comment]
@jkomyno Thank you!

**Comment by jkomyno** [comment]
@AlimovSV check it out

**Comment by erikras** [comment]
Published fix in [`v7.1.0`](https://github.com/erikras/redux-form/releases/tag/v7.1.0).
